R/projpred.R

```diff
@@ -21,63 +26,25 @@
 #'            data = epilepsy, family = poisson())
 #' summary(fit)
 #' 
-#' # perform variable selection without cross-validation
-#' vs <- varsel(fit)
-#' summary(vs)
-#' plot(vs)
-#' 
-#' # perform variable selection with cross-validation
-#' cv_vs <- cv_varsel(fit)
-#' summary(cv_vs)
-#' plot(cv_vs)
+#' # This 'if' condition is only needed here to ensure package 'projpred'
+#' # exists:
+#' if (requireNamespace("projpred", quietly = TRUE)) {
```

Comment:
Can we reformat this to install projpred is not yet installed (or add a comment that the code below only runs with projpred installed so we do not have to use the "if" statement which I don't think is so pretty for an example.

Reply:
Sure, we can change this. Sorry, I didn't realize that the `\dontrun{}` statement will prevent that code from being executed anyway. I'll push a fix.